airflow/models/serialized_dag.py

```diff
@@ -212,3 +228,18 @@ def bulk_sync_to_db(dags: List[DAG], session=None):
                     min_update_interval=MIN_SERIALIZED_DAG_UPDATE_INTERVAL,
                     session=session
                 )
+
+    @classmethod
+    @provide_session
+    def get_last_updated_date(cls, dag_id: str, session: Session = None) -> datetime:
+        """
+        Get the date when the Serialized DAG associated to DAG was last updated
+        in serialized_dag table
+
+        :param dag_id: DAG ID
+        :type dag_id: str
+        :param session: ORM Session
+        :type session: Session
+        """
+        result = session.query(cls.last_updated).filter(cls.dag_id == dag_id).one()
+        return result.last_updated
```

Comment:
```suggestion
        return session.query(cls.last_updated).filter(cls.dag_id == dag_id).scalar()
```

Reply:
Updated in https://github.com/apache/airflow/pull/9851/commits/02856d63a5d500788c8aab7c49a235b036800364